src/style.css

```diff
@@ -0,0 +1,97 @@
+html {
+  font-family: "Roboto", sans-serif;
+  --main-accent-color: #060b35;
+  --secondary-color: #616070;
+  --blue-accent: #00acdc;
+}
+
+body,
+h2,
+p {
+  margin: 0;
+
+}
+
+.product-card {
+  width: 200px;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product-card__image {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  width: 160px;
+  height: 134px;
+}
+
+.product-card__name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--main-accent-color);
+
+  padding-top: 40px;
+}
+
+.product-card__code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--secondary-color);
+
+  padding-top: 4px;
+}
+
+.product-card__price-container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  padding-top: 24px;
+}
+
+.product-card__text {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--secondary-color);
+}
+
+.product-card__price {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--main-accent-color);
+}
+
+.product-card__button {
+  display: block;
+  width: 166px;
+  height: 40px;
+  margin-top: 16px;
+  box-sizing: border-box;
+
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 40px;
+  text-decoration: none;
+  text-transform: uppercase;
+  text-align: center;
+
+  color: #fff;
+  background-color: var(--blue-accent);
+  border: 1px solid transparent;
+  border-radius: 5px;
+
+  transition: color 300ms, background-color 300ms, border-color 300ms;
+}
+
+.product-card:hover .product-card__button {
+  color: var(--blue-accent);
+  background-color: #fff;
+  border-color: var(--blue-accent);
+}
```

Comment:
here I don't understand why you put hover on all product cards. I think we need to do it just for the button `product-card__button: hover ` will be enough

Reply:
Agree with you. I confused it with the case when there is no button and the whole block is link.